onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlDFT.h

```diff
@@ -573,26 +905,38 @@ class GpuDFTOperator : public WRL::Base<IMLOperatorKernel>
         return dims;
     }
 
+    template <typename TConstants, uint32_t TSize>
```

Comment:
wouldn't something like `resourceCount` be better here? `TSize` would be what we want to name the type of a variable, but in this case we use `TSize` as an actual variable.